inference-engine/src/mkldnn_plugin/cpu_memory_desc_utils.h

```diff
@@ -36,39 +40,32 @@ class MemoryDescUtils {
     static MKLDNNMemoryDesc convertToMKLDNNMemoryDesc(const BlockedMemoryDesc& desc);
 
     /**
-     * @brief Converts InferenceEngine::TensorDesc to MKLDNNMemoryDesc
+     * @brief Converts InferenceEngine::TensorDesc to OnednnBlockedMemoryDesc
      * @param desc InferenceEngine::TensorDesc to be converted
-     * @return converted MKLDNNMemoryDesc
+     * @return converted OnednnBlockedMemoryDesc
      */
-    static MKLDNNMemoryDesc convertToMKLDNNMemoryDesc(const InferenceEngine::TensorDesc& desc);
+    static OnednnBlockedMemoryDesc convertToOnednnBlockedMemoryDesc(const InferenceEngine::TensorDesc& desc);
 
     /**
-     * @brief Converts MemoryDesc to BlockedMemoryDesc
+     * @brief Converts MemoryDesc to CpuBlockedMemoryDesc
      * @param desc MemoryDesc to be converted
-     * @return converted BlockedMemoryDesc
+     * @return converted CpuBlockedMemoryDesc
      */
-    static BlockedMemoryDesc convertToBlockedDescriptor(const MemoryDesc& desc);
+    static CpuBlockedMemoryDesc convertToCpuBlockedDescriptor(const MemoryDesc& desc);
 
     /**
-     * @brief Converts MKLDNNMemoryDesc to BlockedMemoryDesc
+     * @brief Converts MKLDNNMemoryDesc to CpuBlockedMemoryDesc
      * @param desc MKLDNNMemoryDesc to be converted
-     * @return converted BlockedMemoryDesc
-     */
-    static BlockedMemoryDesc convertToBlockedDescriptor(const MKLDNNMemoryDesc& inpDesc);
-
-    /**
-     * @brief Creates MKLDNNMemoryDesc with offset0 of UNDEFINED_DIM size
-     * @param desc modifiable MKLDNNMemoryDesc
-     * @return pointer to MKLDNNMemoryDesc
+     * @return converted CpuBlockedMemoryDesc
      */
-    static MemoryDescPtr applyUndefinedOffset(const MKLDNNMemoryDesc& desc);
+    static CpuBlockedMemoryDesc convertToCpuBlockedDescriptor(const OnednnBlockedMemoryDesc& inpDesc);
 
     /**
-     * @brief Creates BlockedMemoryDesc with offsetPadding, strides of UNDEFINED_DIM size and offsetPaddingToData of 0 size
+     * @brief Creates BlockedMemoryDesc with offsetPadding of UNDEFINED_DIM size
      * @param desc modifiable BlockedMemoryDesc
-     * @return pointer to BlockedMemoryDesc
+     * @return pointer to MemoryDesc
      */
-    static MemoryDescPtr applyUndefinedOffset(const BlockedMemoryDesc& desc);
+    static MemoryDescPtr applyUndefinedOffset(const MemoryDesc* desc);
```

Comment:
Why did you changed it to the raw pointer. I think `(const MemoryDesc& desc)` would be better in terms of mem safety.